rstudio/rstudio
# deprecate the Tools / Shell command

### Intent

Addresses #11253 (remove the command for launching native system shell via Tools / Shell)

### Approach

- Got rid of Tools / Shell command (from both desktop and server) and cleaned up the related code
- Substituted the "New Terminal" command for the Shell command on the version control (git/svn) gear menu:

<img width="552" alt="Screen Shot 2022-06-21 at 14 21 42" src="https://user-images.githubusercontent.com/10569626/174890788-9c6a340b-f369-4afb-b0e0-661bf28234fa.png">

### Automated Tests

- There is a selenium test that looks for this menu; I will remove that before merging this PR.

### QA Notes

- Verify that Tools / Shell no longer appears in menus on Desktop and Server/Workbench
- Verify that New Terminal is shown on the Git/Svn dropdown instead of Shell
- Verify that "Open Shell" command no longer appears in the Command Palette

### Checklist
